src/results/wind.jl

```diff
@@ -69,3 +69,64 @@ function add_wind_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
 	d[t] = r
     nothing
 end
+
+"""
+MPC `Wind` results keys:
+- `electric_to_storage_series_kw`
+- `electric_to_grid_series_kw`
+- `electric_curtailed_series_kw`
+- `electric_to_load_series_kw`
+- `electric_to_electrolyzer_series_kw`
+- `electric_to_compressor_series_kw`
+- `energy_produced_kwh`
+"""
+function add_wind_results(m::JuMP.AbstractModel, p::MPCInputs, d::Dict; _n="")
+
+	r = Dict{String, Any}()
+	t = "Wind"
+	if !isempty(p.s.storage.types.elec) 
+		WindtoBatt = (sum(m[Symbol("dvProductionToStorage"*_n)][b, t, ts] for b in p.s.storage.types.elec) for ts in p.time_steps)
+		WindtoBatt = round.(value.(WindtoBatt), digits=3)
+	else
+		WindtoBatt = zeros(length(p.time_steps))
+	end
+	r["electric_to_storage_series_kw"] = WindtoBatt
+
+	if !isempty(p.s.electric_tariff.export_bins)
+		WindtoGrid = @expression(m, [ts in p.time_steps],
```

Comment:
I've seen issues with results being expressions and thus the type is JuMP.Containers.DenseAxisArray instead of vector